test/regression/riscv/InorderPipeline.cc

```diff
@@ -0,0 +1,38 @@
+#include "RISCVRegressionTest.hh"
```

Comment:
This may be better suited in the `unit` test suite

Reply:
I did think about this but not sure of the best way to do it as it needs all of the ```RISCVRegressionTest.cc``` etc. infrastructure to run

Reply:
Noted, leave as is